sentry-ruby/lib/sentry/client.rb

```diff
@@ -54,31 +39,16 @@ def capture_event(event, scope)
       event
     end
 
-    def event_from_exception(exception, **options)
-      exception_context =
-        if exception.instance_variable_defined?(:@__sentry_context)
-          exception.instance_variable_get(:@__sentry_context)
-        elsif exception.respond_to?(:sentry_context)
-          exception.sentry_context
```

Comment:
Note how `#sentry_context` used to be able to provide every aspect of event context, not just `extra`.